app/subscribers/foreman_rh_cloud/insights_subscriber.rb

```diff
@@ -0,0 +1,8 @@
+module ForemanRhCloud
+  class InsightsSubscriber < ::Foreman::BaseSubscriber
+    def call(*args)
+      host = args.first.payload[:object]
+      host.get_status(InsightsClientReportStatus).refresh!
```

Comment:
This causes non-managed hosts with `host_registration_insights` param to be displayed as "Reporting" even though the host is not provisioned at all.